src/util.ts

```diff
@@ -332,21 +335,19 @@ const slugifyMethods: { readonly [mode in SlugifyMode]: (rawContent: string) =>
     /**
      * GitLab
      */
-    "gitlab": (slug: string): string => {
+    [SlugifyMode.GitLab]: (slug: string): string => {
```

Comment:
Did you forget to do it for other slugify modes?

Reply:
No. I just did not want to create a too long diff.

Then, let's make them consistent.